README.md

```diff
@@ -58,8 +62,11 @@ Further steps are described more in Workflow section.
 
 ## Workflow
 
+
+// REVIEW-VINI: This image is broken on my machine, you can try to use mermaid to create diagrams - Potential fix is: architecture/Order%20flow.jpg 
```

Comment:
I can see mermaid advantages in creating diagrams, but imo it's time consuming and has its own limitations. I changed image links to relative paths, so now they should appear fine in IDE as well